core/bin/via_server/src/node_builder.rs

```diff
@@ -358,6 +359,42 @@ impl ViaNodeBuilder {
         Ok(self)
     }
 
+    fn add_commitment_generator_layer(mut self) -> anyhow::Result<Self> {
+        self.node.add_layer(CommitmentGeneratorLayer::new(
+            self.genesis_config.l1_batch_commit_data_generator_mode,
+        ));
+
+        Ok(self)
+    }
+
+    fn add_da_dispatcher_layer(mut self) -> anyhow::Result<Self> {
+        let eth_sender_config = try_load_config!(self.configs.eth);
+        if let Some(sender_config) = eth_sender_config.sender {
```

Comment:
Ok for now, but lets not forget to remove `EthConfig` usage and move the required config to some Via related one.

Reply:
actually we don't need this check, cause we only have validium mode in btc_sender, so we can remove this check.